app/models/user_rate.rb

```diff
@@ -0,0 +1,12 @@
+class UserRate < ActiveRecord::Base
+  # :belongs_to :user
+  belongs_to :book
+
+  RATES = {
+    1 => "Very Bad",
+    2 => "Bad",
+    3 => "Medium",
+    4 => "Good",
+    5 => "Very Good"
+  }.freeze
```

Comment:
I'd add validation to be sure that it's impossible to create UserRate with rate outside of RATES range :)